index.html

```diff
@@ -291,7 +290,7 @@ <h2 id='stipends'>Stipends</h2>
                 with a short letter of support.
                 </p>
 
-                <h2 id='submissions'>Submissions</h2>
+                <h2>Submissions</h2>
```

Comment:
sure - all suggestions are good with me

please change and post

On Fri, Mar 25, 2016 at 7:47 PM, Dionysis Zindros <notifications@github.com>
wrote:

> In index.html
> <https://github.com/dionyziz/bitcoinschool/pull/6#discussion_r57503177>:
>
> > @@ -291,7 +290,7 @@ <h2 id='stipends'>Stipends</h2>
> >                  with a short letter of support.
> >                  </p>
> >
> > -                <h2 id='submissions'>Submissions</h2>
> > +                <h2>Submissions</h2>
>
> Doesn't harm to keep the id in case people want to link to it – just let's
> not bloat the menu
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dionyziz/bitcoinschool/pull/6/files/41aa1f6d7870824b5c8bbc6c9c797f8abf0a69e5..8443f494fa425309d762232b0ad3f87b25784e6a#r57503177>
>